src/System.Slices/System/ReadOnlySpan.cs

```diff
@@ -333,5 +343,9 @@ public override bool Equals(object obj)
             }
             return false;
         }
+
+        public bool Equals(T[] other) => Equals(new ReadOnlySpan<T>(other));
```

Comment:
Yeah, I guess my question was if `IEquatable<ReadOnlySpan<T>>` needed to be implemented...